packages/react/error-boundary/src/ErrorBoundary.en.md

```diff
@@ -0,0 +1,37 @@
+# ErrorBoundary
+
+- see https://jbee.io/react/error-declarative-handling-1/ Declaratively handle loading and error conditions
+- see https://toss.im/slash-21/sessions/3-1 Suspense & error handling presentation in Slash 21
+
+Declaratively, the component used to manage errors.
+`ErrorBoundary` component catches errors in the render/useEffect and renders the given component with `renderFallback`
+
+```typescript
+<ErrorBoundary
+  // Components to be rendered when an error occurs.
+  // The first parameter represents a caught error.
+  renderFallback={error => <div>error is occured. {error.message}</div>}
+  /*
+    Callback function to be called when an error occurs.
+    The first parameter represents the caught error, and the second parameter represents the stack of the
+    component where the error occurred.
+    Type of componentStack의 is `string`
+  */
+  onError={(error, { componentStack }) => {
+    alert(error.message);
+    console.log(componentStack);
+  }}
+  // If the value in the array changes, reset the error caught by ErrorBoundary.
+  // Verify that the values are the same with `Object.is()`.
+  // @default []
```

Comment:
Existing comments have been listed one by one using `//` keywords, even if they are multi-line , but if they are multi-line comments, how about using the `/*` keyword? I think it's more readable.

Reply:
I think `//` and `/*` have the same readability, so it's just a convention.
I think `//` is more readable because it can separate a long paragraph into one clear sentences.

Reply:
>  it can separate a long paragraph into one clear sentences.
thank you! I keep the convention

Reply:
[fix: /* => // for multiple sentence](https://github.com/toss/slash/pull/102/commits/d6c9ca5526b39292dce1336101d92c882e77cf16)
[d6c9ca5](https://github.com/toss/slash/pull/102/commits/d6c9ca5526b39292dce1336101d92c882e77cf16)